frontend/javascripts/admin/auth/login_view.tsx

```diff
@@ -30,7 +30,7 @@ function LoginView({ history, redirect }: Props) {
   return (
     <Row justify="center" align="middle" className="login-view">
       <Col xs={22} sm={20} md={16} lg={12} xl={8}>
-        <Card className="login-content">
+        <Card className="login-content" style={{ margin: "0 auto" }}>
```

Comment:
Die `row` in Zeile 31 ist `centered` doch schon den content. Ich versteh das nicht?

Reply:
The row centers its children, thats correct. Thus, the column is centered. But the Card (child of the column)  does not fill the columns full width in case of large screens. In that scenario the Card just starts at the left edge of the column and is therefore not centered on large screens. Thist margin auto fix, mitigates this and the result is a centered login view.

Reply:
Right. In that case I would add the `margin: auto` style to the `login-content` CSS class/Less file directly. That way "Sign up" and "Reset Password" also profit from theses changes.